README.md

```diff
@@ -44,3 +44,60 @@ module.exports = {
 ```bash
 yarn jest
 ```
+
+
+## Options
+
+This project uses [cosmiconfig](https://github.com/davidtheclark/cosmiconfig), so you can provide config via:
+* a `jest-runner-eslint` property in your `package.json`
+* a `jest-runner-eslint.config.js` JS file
+* a `.jest-runner-eslintrc` JSON file
+
+
+In `package.json`
+```json
+{
+  "jest-runner-eslint": {
+    "cliOptions": {}
+  }
+}
+```
+
+See https://eslint.org/docs/developer-guide/nodejs-api#cliengine for a full
```

Comment:
I think `cliOptions` should actually map to https://eslint.org/docs/user-guide/command-line-interface

Reply:
I think that is a good point.

Even though it would be more work, because we need to make sure that we do the correct mapping of "pseudo CLI options" to API options, but it will definitely provide a better experience.

After looking at the docs here is the list of CLI options that we could have. It does not include all of them because the API does not accept all of them. Although a lower priority, but we might need to shim the rest

|Proposed option | ESLint CLI option| Notes|
|:---------------|:-----|:-----|
|noInlineConfig | --no-inline-noInlineConfig|
|cache | --cache|
|cacheLocation | --cache-location|
|config | --config|
|envs | --env | _Needs to be pluralized because `--env` can be passed multiple times_
|exts | --ext | _Needs to be pluralized because `--ext` can be passed multiple times_
|fix | --fix|
|globals | --global | _Needs to be pluralized because `--global` can be passed multiple times_
|noIgnore | --no-ingore|
|ignorePath | --ignore-ignorePath|
|parser | --parser|
|parserOptions | --parser-parserOptions|
|plugins | --plugin | _Needs to be pluralized because `--plugin` can be passed multiple times_
|rulesdirs | --rulesdir | _Needs to be pluralized because `--rulesdir` can be passed multiple times_
|rules | --rule | _Needs to be pluralized because `--rule` can be passed multiple times_
|noEslintrc | --no-eslintrc|
|exts | --ext  | _Needs to be pluralized because `--exts` can be passed multiple times_


What do you think?

Reply:
I don't think they need to be pluralized; I think if they want more than one, it can be an array - like `ext: 'foo'` or `ext: ['foo', 'bar']`